### DISCUSSION TITLE
service Active: inactive (dead)

### DISCUSSION THREAD

**Discussion by fanlessfan** [bluenviron/mediamtx#255] [Questions]
I downloaded the arm6l binary from release and put it in raspberry pi 0. I use the server file below to start rasp-simple-server in systemd in. but it always shows inactive. I can use systemctl restart rasp_server to successfully start the service. I did the same thing in raspberry pi 3B and everything is OK. 

Anyone can help?

thx

sudo systemctl status rtsp_server
● rtsp_server.service - rtsp_server
   Loaded: loaded (/etc/systemd/system/rtsp_server.service; disabled; vendor preset: enabled)
   Active: inactive (dead)

[Unit]
Description=rtsp_server
After=network.target rc-local.service
[Service]
Restart=always
RestartSec=1s
User=pi
ExecStart=/usr/local/bin/rtsp-simple-server
[Install]
WantedBy=multi-user.target

**Comment by aler9** [comment]
Hi, i tried your systemd file in a Raspberry pi zero, and it does work without problems.
Did you enable the service at startup time? (`systemd enable rtsp_server`)
You can also check the logs (`journalctl -xeu rtsp_server`)
I usually do not use Restart and RestartSec, since i want a service to fail if there's any problem.

**Comment by aler9** [comment]
Systemd instructions have been added to the README:
https://github.com/aler9/rtsp-simple-server#start-on-boot-with-systemd

[a comment by an automated account is not shown]
